samples/settings/src/main.c

```diff
@@ -61,6 +62,14 @@ int main(void)
 		net_connect();
 	}
 
+	/* Note: In production, you would provision unique credentials onto each
+	 * device. For simplicity, we provide a utility to hardcode credentials as
+	 * kconfig options in the samples.
+	 */
+	if (IS_ENABLED(CONFIG_GOLIOTH_SAMPLE_HARDCODED_CREDENTIALS)) {
+		hardcoded_credentials_set();
+	}
+
```

Comment:
How about calling `hardcoded_credentials_set()` automatically during boot by `SYS_INIT(..., APPLICATION, ...)`? That way we would not need to have that as part of every sample and would keep sample specific code minimal.

Reply:
I like it!